tests/karate/src/test/java/be/constants.feature

```diff
@@ -2,34 +2,55 @@
 Feature: Constants
   Scenario: Creates constants that will be used by other features
     # TODO: make the function depend on all the attributes the lao id depends on
-    * def createLaoIdEmptyName =
-      """
-        function(){
-          return "p8TW08AWlBScs9FGXK3KbLQX7Fbgz8_gLwX-B5VEWS0="
-        }
-      """
-    * def createLaoIdNegativeTime =
-      """
-        function(){
-          return "p8TW08AWlBScs9FGXK3KbLQX7Fbgz8_gLwX-B5VEWS0="
-        }
-      """
-    * def createLaoValid =
+    * def organizerPk =
       """
-        function(){
-          return "p_EYbHyMv6sopI5QhEXBf40MO_eNoq7V_LygBd4c9RA="
-        }
+        function(){ return "J9fBzJV70Jk5c-i3277Uq4CmeL4t53WDfUghaK0HpeM=" }
       """
-    * def createRollCallValid =
+    * def attendeePk =
       """
         function(){
-          return "Slj7C1LBEXlRC8ItV2B0zWfUSD6YiGJt6N_I_m02uw4="
+          return "M5ZychEi5rwm22FjwjNuljL1qMJWD2sE7oX9fcHNMDU="
         }
       """
-    * def createRollCallInvalid =
-      """
-        function(){
-          return "Dui7C1LBEXlRC8ItV2B0zWfUSD6YiGJt6N_I_m02uw4="
+
+    * def getOrganizer = call organizerPk
+    * def getAttendee = call attendeePk
+
+    * def laoValidCreationTime = function(){ return 1633035721 }
```

Comment:
What does this time corresponds to ?
Beware that in the future, backends might add checks on the validity of this value. (Not too far in the past nor in the future)

Maybe add a small comment explaining that.